core/client/src/main/java/alluxio/client/file/FileSystemUtils.java

```diff
@@ -145,59 +135,29 @@ public static boolean waitCompleted(final FileSystem fs, final AlluxioURI uri,
    *
    * @param fs {@link FileSystem} to carry out Alluxio operations
    * @param uri the uri of the file to persist
-   * @param status the status info of the file
-   * @return the size of the file persisted
    * @throws IOException if an I/O error occurs
    * @throws FileDoesNotExistException if the given file does not exist
    * @throws AlluxioException if an unexpected Alluxio error occurs
    */
-  public static long persistFile(FileSystem fs, AlluxioURI uri, URIStatus status)
-      throws IOException, FileDoesNotExistException, AlluxioException {
-    // TODO(manugoyal) move this logic to the worker, as it deals with the under file system
-    Closer closer = Closer.create();
-    long ret;
+  public static void persistFile(final FileSystem fs, final AlluxioURI uri)
```

Comment:
Yes, this is covered by various test cases in `PersistCommandTest`, such as  `alluxio.shell.command.PersistCommandTest#persistWithAncestorPermission`